source/jquery.flot.touch.js

```diff
@@ -275,10 +271,12 @@
             return false;
         }
 
-        function preventEventPropagation(e) {
-            if (!gestureState.allowEventPropagation) {
+        function preventEventDefault(e) {
```

Comment:
Can we call this `preventEventBehaviors` as this can still do something more than just prevent the default.

Reply:
Renamed.